db/structure.sql

```diff
@@ -39847,10 +39839,10 @@ CREATE UNIQUE INDEX "index_sys.cdr_exports_on_uuid" ON sys.cdr_exports USING btr
 
 
 --
--- Name: network_prefixes_prefix_range_idx; Type: INDEX; Schema: sys; Owner: -
+-- Name: network_prefixes_prefix_with_scope_idx; Type: INDEX; Schema: sys; Owner: -
 --
 
-CREATE INDEX network_prefixes_prefix_range_idx ON sys.network_prefixes USING gist (((prefix)::public.prefix_range));
```

Comment:
prefix_range index by prefix shouldn't be modified. We have only change UNIQUE  index.